students/JeffBennett/Lesson04/file_lab.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+import os
+import pathlib
+
+# Activity 2: File lab
+print('\nFile Lab\n')
+
+# prints full path name
+for file in os.listdir():
+    print(os.getcwd() + '/' + file)
```

Comment:
avoiding using `file` for var name, it's a built-in
also, better to use `os.path.join` for creating file paths